### ISSUE TITLE
JSPM bundling fails with spaces in path

### ISSUE THREAD

**Issue by JeroenVinke** [systemjs/builder#235]
JSPM bundling works when using a path without spaces, and fails with spaces.

I am using Aurelia, which has a path mapping in config.js: ` "*": "src/*.js",`
And I got this error when bundling `*`: `err  Error: ENOENT, open 'D:\Development\test a\src\src\app.js.js'`

Notice the space between `test a` in my path.

I have logged `normalized` and `curPath` in `getCanonicalName` and this was the output:

```
normalized: file:///D:/Development/test a/src/app.js
curPath:    file:///D:/Development/test%20a/src/*.js
```

When getCanonicalName is doing path comparisons it fails.
Bundling starts working again when I replace %20 by a space in `curPath`, or space by %20 in `normalized`

@ahmedshuhel

**Comment by JeroenVinke** [comment]
Workaround for Windows is mapping a drive letter to the folder:

```
subst x: .
x:
jspm bundle *
d:
subst x: /D
```

**Comment by guybedford** [comment]
Thanks so much for the detailed report here, I've posted a fix in https://github.com/systemjs/builder/commit/e74066f92629e4287352bb31c475990e856930e0 - the issue is that `url.resolve` in Node does URI encoding as well.

**Comment by guybedford** [comment]
Released in 0.13.0.
